### PR TITLE
A new SLE 12/15 rule ensure_package_repositories_are_configured/

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _This rule covers CIS requirement 1.2.2 Ensure package manager repositories are configured_

#### Rationale:

- At the moment CIS profiles for SLE 12/15 does not include this recommendation. The control cannot be automated, 
       and should be addressed manually.

### CODE REVIEW

File: linux_os/guide/system/software/updating/ensure_package_repositories_are_configured/rule.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+prodtype: sle12,sle15
+
+title: 'Ensure package manager repositories are configured'
+
+description: |-
+    Systems need to have package manager repositories configured to ensure they receive the
+    latest patches and updates.
+
+rationale: |-
+    If a system's package repositories are misconfigured important patches may not be
```
Comment (marcusburghardt) [inline linux_os/guide/system/software/updating/ensure_package_repositories_are_configured/rule.yml:12]:
```suggestion
    If the system's package repositories are misconfigured important patches may not be
```

File: linux_os/guide/system/software/updating/ensure_package_repositories_are_configured/rule.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-92389-6
+    cce@sle15: CCE-92489-4
+
+references:
+    cis@sle12: 1.2.2
+    cis@sle15: 1.2.2
+
```
Comment (marcusburghardt) [inline linux_os/guide/system/software/updating/ensure_package_repositories_are_configured/rule.yml:24]:
```suggestion
```

**Comment by marcusburghardt** [comment]
@rumch-se , this seems to be missing only some minor updates and should be ready to be merged. Could you take a look, please?

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
Thank you for your feedback.
I have made the proposed corrections.
Have a nice day
Rumen

**Comment by marcusburghardt** [comment]
`Automatus CS8` and `Automatus CS9` are failing because this rule new rule is restricted to `prodtype: sle12,sle15`.

**Review by marcusburghardt — APPROVED** [review]
Thanks @rumch-se

**Comment by marcusburghardt** [comment]
Overriding the CODEOWNERS protection since there is no SUSE contributor with write permission at the moment.

**Pull request merged**